### ISSUE TITLE
PC restarts without warning while Multi-GPU training

### ISSUE THREAD

**Issue by phonygene** [AlexeyAB/darknet#2731]
OS : win7
GPU : RTX2080ti x 2
Driver Version : 419.35 CUDA Version: 10.1
PSU : 1200w (CoolerMaster V1200 Platinum)
Using master-branch downloaded on 2019/03/05.

Sorry for bugging you again.
Got restart without warning while training with two gpus ,
`darknet.exe detector train face.data yolov3.cfg yolov3_1000.weights -gpus 0,1`

My first thought is the issue of PSU.
So I do the test with commands:
`darknet.exe detector train face.data yolov3.cfg yolov3_1000.weights -gpus 0`
`darknet.exe detector train face_b.data yolov3_b.cfg darknet53.conv.74 -gpus 1`
I kept it running a weekend and it went well.
But every time I tried to train with  `-gpus 0,1`  , my pc will restarts within 5-30 minutes.

Is it possibly because of memory conflict ?
Or is this more likely to be a hardware problem?

**Comment by AlexeyAB** [comment]
@phonygene Hi,

This is the most likely a hardware or driver problem.

User-space software problem can't lead to PC rebooting.

Try to increase `subdivisions=` 2x times more and train again.

Try to run GPU-Z and look does your GPU use Micron GPU-RAM?
There were some hardware problems with it:
* https://www.tomshardware.com/news/rtx-2080-ti-gpu-defects-launch,37995.html
* https://www.reddit.com/r/nvidia/comments/ac3zj2/rtx_2080_ti_micron_users_still_going_good/

**Comment by phonygene** [comment]
@AlexeyAB :

Thanks for the information. 
I have done some more tests. I tried to keep the GPU card at full load as possible.
It turns out that restart also happens when two GPUs training different models.
But if I only activate one GPU at a time, then it just works fine, whether with yolo or burn-in tool like FurMark.

It seems the PSU is the most suspicious one.
So I contacted CM and exchanged v1300 with v1200 ,
now the problem solved.

In conclusion: 

1200W PSU may not powerful  enough for two 2080ti GPUs training.
My 2080ti Turbo cards which are made with micron memory just working well.
